docs/examples/chat_engine/chat_engine_c3.ipynb

```diff
@@ -0,0 +1,512 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    ] = "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"
</pre><p>better to remove your api key</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
yeah that was my bad .. I invalidated it from the account immediately, but will also clear it from here so as not to make others hopeful :D ..